### PR TITLE
Start the migration, bitch

### PR DESCRIPTION
**Pull request by RoySegall**
#146

### CODE REVIEW

**Comment by nirgn975** [comment]
OMG I love the title!

**Comment by RoySegall** [comment]
OK, YL is also formatted OK... Kind of. It for some reason adds `,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,` in the end. I need to find out why.

**Comment by RoySegall** [comment]
@nirgn975 We need at this point add QA to this branch.

**Comment by nirgn975** [comment]
@RoySegall Let's start this tomorrow, and we have the Hackathon next week.

**Comment by RoySegall** [comment]
I'm doing the weird string manipulation because of: 
evaluate_expression_and_pluginbase_py_-_open_pension_-___applications_mamp_htdocs_open_pension_

**Comment by RoySegall** [comment]
Three things to handle:
1. `yl_2016_3_9941/פקדונות מעל 3 חודשים-Table 1.csv` - handle "110,000.00-" and check if that broke the line,
2. Add the metadata file to the library
3. Handle the tests import things

**Comment by RoySegall** [comment]
Before:
yl_2016_3_9941_-_google_drive

After:
yl_2016_3_9941_-_google_drive

File: server/pension/management/commands/plugins/PluginBase.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        if row_context:
+            # Get the current context.
+            self.local_context = command.english_text(row_context)
+
+            # Check if the current context is a global context or.
+            if command.is_global_context(self.local_context):
+                self.global_context = self.local_context
+        else:
+            value = self.getCleanRow(value)
+            value = value + "," + self.global_context + "," + self.local_context
```
Comment (nirgn975) [inline server/pension/management/commands/plugins/PluginBase.py]: `value = value + "," + self.global_context + "," + self.local_context` => `'{value},{global_context},{local_context}'.format(value=value, global_context=self.global_context, local_context=self.local_context)`

**Pull request merged**